docs/UserCommands.md

```diff
@@ -0,0 +1,44 @@
+## User Command Reference
+
+
+`@rocket user`
+
+## Options to specify input:
+
+* `-edit`
+    * `-name` somename `-email` email `-pos` yourposition `-github` githublink `-major` yourmajor `-bio` yourbio
+        * edit properties of your Launch Pad profile
+    * ADMIN only option: `-member` MEMBER_ID
+        * edit properties of another user's profile
+
+
+* `-view` MEMBER_ID
+    * view information about a user
+
+
+* `-add`
+     * `-team` TEAM_ID `-member` MEMBER_ID
+        * add user to team
+
+
+* `-rmv`
```

Comment:
I would prefer this to be `remove` - it's more characters to type, but it'll be harder for people to remember the abbreviation than the full word.